config_parse.go

```diff
@@ -28,6 +30,11 @@ func ReadProxyConfig(path string) (c ProxyConfig, err error) {
 		return
 	}
 
+	err = envconfig.Process("veneur", &c)
+	if err != nil {
+		log.Fatal(err.Error())
```

Comment:
We should bail and let them know it didn't work, but since we can return an error, we should do that instead of calling `log.Fatal` immediately. At the very least, that means that we can test this error condition without aborting the entire test suite. :)